tools/server/webui/src/routes/+layout.svelte

```diff
@@ -139,7 +139,7 @@
 
 <ModeWatcher />
 
-<Toaster richColors />
+<Toaster position="top-center" richColors />
```

Comment:
Not sure about this... I feel that the toaster notifications are much better in their own space at their default bottom right corner as there's no other UI that they might be obfuscating when it's not necessary.
```suggestion
<Toaster richColors />
```